backend/services/iot-manager/api/http/management_state.go

```diff
@@ -28,8 +28,9 @@ import (
 )
 
 const (
-	paramDeviceID      = "id"
-	paramIntegrationID = "integrationId"
+	paramDeviceID           = "id"
+	paramIntegrationID      = "integrationId"
```

Comment:
This parameter is unused now, right?

Reply:
Sorry, I was confused by the indentation change.

Reply:
do not despair! it is going to be alright. 😘